src/index.js

```diff
@@ -0,0 +1,52 @@
+import './styles.scss';
+import reload from './assets/refresh.png';
+import enter from './assets/enter.png';
+import addList from './addList.js';
+import List from './List.js';
+import Store from './store.js';
+import dummyList from './dynamicObj.js';
+
+const heading = document.getElementById('reloadImg');
+heading.src = reload;
+
+document.getElementById('addBtn').innerHTML = `
+<img src=${enter} alt="enter" />
+`;
+
+document.getElementById('form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const input = document.getElementById('input').value;
+  if (input === '') {
+    addList.showAlert();
+  } else {
+    const list = new List(input);
+    addList.addList(list);
+    Store.addList(list);
+    addList.clearFields();
+  }
+});
+
+const removeElem = () => {
+  document.querySelectorAll('.list-group-item').forEach((elem) => {
+    elem.querySelectorAll('.form-check-input').forEach((item) => {
+      if (item.checked === true) {
+        item.parentNode.parentNode.remove();
+        addList.removeCompleted(item);
+        Store.removeList(item);
+      }
+    });
+  });
+};
```

Comment:
- [ ] [OPTIONAL] Please note that for this milestone, all that is required is the to-do list structure and not the full blown application. The add, update, remove, and clear-all-completed to-do items features will be added in subsequent milestones. The previous reviewer did not point this out and that's why it is OPTIONAL, but I strongly suggest that you remove those features before you merge 👍